### DISCUSSION TITLE
巴黎奥运会—包租婆单手插兜力争冠军！

### DISCUSSION THREAD

**Discussion by diviner0103** [share121/inter-knot#1498] [其他]
![1723132960272](https://github.com/user-attachments/assets/5b1741cc-c508-494e-89c8-2bdaf4825a69)
不知道大家最近有没有关注巴黎奥运会，今天比赛包租婆单手插兜比赛，简直帅翻了，实力超强，有望为中国队再夺得一金！

**Comment by Kamisato100** [comment]
味对了

**Comment by Philia093-T** [comment]
看了，前几天都被她刷屏了

**Comment by wsskhop** [comment]
6

**Comment by laoyee234** [comment]
在现场，我是那根烟

**Comment by pjpPHARAOHpharaoh** [comment]
是的

**Comment by 1451374815** [comment]
虚构史学家

**Comment by Phaethon-0019** [comment]
包租婆有对手了，之前看比赛的时候有个选手记不清哪个国家了，只知道是西欧的一个国家，叫尚勃勒，气步枪气手枪成绩都不错的，那场比赛估计很精彩

**Comment by QsLingYin** [comment]
太对了

**Comment by Eurus1997** [comment]
我觉得差一点，霓虹国有个名叫山上彻也的劲敌，战绩可查的，不知道包租婆顶不顶得住。<img src="https://bbs-static.miyoushe.com/static/2023/12/19/190b724e523a106f66501b78daded09d_1307672139121863195.png" alt="真理医生 裂开" width="75" >

**Comment by Rolesican** [comment]
不是，你们看到美国选手了吗，纯摆子，rnm!退钱！！
